CONFIGURATION.md

```diff
@@ -18,8 +18,12 @@ Your configuration overrides the default configuration, which can be found
   // If set to 0, no filter will be applied
   "maxAge": 168,
   
-  // How many projects will be fetched from GitLab.
+  // How many projects will be fetched from Gitlab. Usually you want all projects under a user/group.
```

Comment:
- Gitlab → GitLab
- In don't understand the differences between these two options. :sweat_smile:

Reply:
Unsure if it makes sense for a projectScope `user`, but it makes sense for `group` and used together with include_subgroups. Especially since the application currently is hammering gitlab with requests. 

Further pull requests will arrive which will use a  feature flag for reducing API calls, in loss of delayed updates.

Reply:
@timoschwarzer pushed the typo fix, an idea for better comment about "Usually you want all projects under a user/group"? Or is it fine?

Reply:
I still don't get the difference between fetchCount and fetchCountProjects.

Reply:
@timoschwarzer : `fetchCountProjects` is the limit used towards GitLab API when you are polling which projects are available for our group. We usually want all projects to be shown on our dashboard, but we don't want it to pull for >20 branches and tags as it amplifies the amount of requests going towards GitLab.

Reply:
Aaah, now I got you.
I'd leave the current option (fetchCount) as it is and add a new option `branchFetchCount`.

For branches, `branchFetchCount` is used when given, else `fetchCount`. If I'd merge this right now we would alter the current behavior which I don't want to without notice.

Reply:
both `pipelines`, `branches` and `tags` (have to verify if more) uses the `fetchCount`, hence I introduced the new one only for `projects` as `fetchCount` were global for every other API call. We can of course add specific fetch count for every type of fetches?

Reply:
The problem is, that if I've currently set `fetchCount` to 20 and I'd silently update to your version, I'd get more than 20 projects because your default is 101.

Reply:
I don't mind changing the default for projects to be 20 as well.

Just decide between if you want a fetchCounter specific for each type of fetch, or one shared one + the customized one for projects fetching or what you want, and I'll do the changes.